frontend/src/components/FooTouer/FooTouer.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './FooTouer.css';
```

Comment:
Znacie style-components? Proponowałbym to zamiast plików css. Co sądzicie :)?

Reply:
Słyszałam coś o tym. Możemy zastosować jeżeli uważasz, że warto